### PR TITLE
Implemented books overview component and styled single book component

### PR DESCRIPTION
**Pull request by adragicevicc**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -2,10 +2,13 @@ import React from 'react';
...
 import App from './App';
+import { BrowserRouter } from 'react-router-dom';
 
 const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement);
 root.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <App />
+    </BrowserRouter>
```
Comment (milos-majkic) [inline src/index.tsx:12]: Oh, I see that you use `<BrowserRouter>` component in the index.tsx so you can ignore my first comment.

File: src/components/BooksList/BooksList.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  const retriveBooks = async () => {
+    try {
+      const response = await axios.get(`http://localhost:8080/book`, config);
+      setBooks(response.data.content);
+    } catch {}
+  };
+
+  // const getBooks = () => {
+  //   setBooks(data);
+  // };
```
Comment (milos-majkic) [inline src/components/BooksList/BooksList.tsx:39]: Please, remove comments.

File: src/variables.scss [hunk 3]
```diff
@@ -20,3 +21,11 @@ $space-s: 0.5rem; // 8px
 $space: 1rem; // 16px
 $space-l: 1.25rem; // 20px
 $space-xl: 1.5rem; // 24px
+
+// Div sizes
+$ds-base: 350px;
+$ds-s: 200px;
```
Comment (vkotarlic) [inline src/variables.scss:27]: I'm not sure what these are, but I'm sure they shouldn't exist 😄

File: src/pages/BooksOverview/BooksOverview.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import BooksList from 'components/BooksList/BooksList';
+import Navbar from 'components/Navbar/Navbar';
+import React from 'react';
+import classes from './BooksOverview.module.scss';
+
+const BooksOverview = () => {
+  return (
+    <div>
+      <Navbar />
```
Comment (vkotarlic) [inline src/pages/BooksOverview/BooksOverview.tsx:9]: We don't want to import `<Navbar />` component in all pages separately, instead you can create Layout component and add `<Navbar />` there, or you can put it in App above router import

File: src/components/BooksList/BooksList.tsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      Authorization:
+        'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJsaWJyYXJ5LmlvIiwiYXVkIjoibGlicmFyeSIsImlzcyI6ImlvLmxpYnJhcnkuYXBwIiwiZXhwIjoxNjY4MDAyNDMxLCJ1c2VySWQiOjEwMDAsImVtYWlsIjoiYWRtaW5AbnFvZGUuaW8iLCJ1c2VyUm9sZSI6IkFETUlOSVNUUkFUT1IifQ.XbaKJrfBP2SH3jQ7jdyDUPMw3su1HFSSo3E9zc-iYbM-er44Tva1I3WTaQURaTBFz8od4SaI6ZgjcfbqQMW7aA'
+    }
+  };
+
+  const [books, setBooks] = useState<BooksListData[]>([]);
+
+  const retriveBooks = async () => {
+    try {
+      const response = await axios.get(`http://localhost:8080/book`, config);
```
Comment (milos-majkic) [inline src/components/BooksList/BooksList.tsx:32]: This is a bad practice to pass link like this.
Please configure .env file to use connect to API from there.

**Review by milos-majkic — APPROVED** [review]

**Pull request merged**